_posts/2019-06-04-bitcoin-and-the-store-of-value-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+But, in this post, we are interested in the bootstrapping process to achieve that salability. Let’s refer back to the stage of goods being just a medium of indirect exchange. I stated above that humans would rely on cereal’s utility as food to foresee its future demand because the concept of medium of indirect exchange would still be an unknown abstraction for them. If we fast forward to a context where humans already conceptually grasp a medium of exchange, why should they not be able to foresee the monetary demand of any good, regardless if it has other uses or not-- that is, its ease of transport, divisibility, durability, and unfalsifiability. This is clearly the case for Szabo’s “collectibles” and also Bitcoin’s case. From a historical point of view, I find it rather unlikely that humans used otherwise useless goods as a medium of exchange without first having discovered the medium of exchange concept through other consumptive goods, but it is certainly theoretically possible. 
+
+In the same way that discovering the concepts of adding and subtracting quantities is more natural through comparing physical quantities (i.e. beads, as in an abacus), that does not necessarily prevent humans to skip that step and directly invent written numerals and ledgers. Even though we once used the abacus and its sexagesimal numeral system, it does not mean that all other numeral systems must first be accounted through the sexagesimal system.. We must not confuse the supporting medium by which we historically experiment and discover a concept (i.e. beads or commodities) with the concept itself (arithmetics or Money).
+
+If something arises (bitcoin) that can fulfill the sole use case of a medium of exchange better than the incumbents (barter, salt, gold, fiat etc.), then that is where the demand can be found. Can this demand can be speculative? Yes, it is speculation on the possibility of the good becoming money in the future.
+
+Mises had the following concern regarding new monetary goods that have no other use than a medium of exchange: (Human Action - Chapter 17 section 4 – “The determination of purchasing power of money”)
+
+*"A value judgment is, with reference to money, only possible if it can be based on appraisement. The acceptance of a new kind of money presupposes that the thing in question already has previous exchange value on account of the services it can render directly to consumption or production. Neither a buyer nor a seller could judge the value of a monetary unit if he had no information about its exchange value--its purchasing power--in the immediate past.”*
```

Comment:
This is still not a blockquote...need to use a `>` symbol for that. Blockquotes are different from regular italic text in that they're indented to make it obvious to the reader that they're blockquotes. No `*` needed.